resources/views/profile/delete-user-form.blade.php

```diff
@@ -32,13 +32,17 @@
                 <form class="mt-8">
                     <div class="space-y-2">
                         <x-ark-input input-class="text-center" type="text" name="username" model="username" :label="trans('fortify::forms.confirm_username')" readonly />
-                        <x-ark-input type="text" name="username_confirmation" model="usernameConfirmation" :palceholder="trans('fortify::forms.delete-user.confirmation_placeholder')" :errors="$errors" hide-label />
+                        <x-ark-input type="text" name="username_confirmation" model="usernameConfirmation" :placeholder="trans('fortify::forms.delete-user.confirmation_placeholder')" :errors="$errors" hide-label />
+                    </div>
+                    <div class="mt-4">
+                        <label for="feedback" class="input-label">@lang('fortify::forms.feedback.label')</label>
```

Comment:
I assume this is done because of the "optional" parameter, but this makes the hover no longer highlight the label. It's better to move the `optional` part to the inputs in the UI lib so it can be properly handled

Reply:
@ItsANameToo Label is highlighted because is linked to the `input` via `for` attribute.
I did it because of the "optional" part. If it's used only here I don't think its worth adding it in the UI lib but let me know..

Reply:
<img width="535" alt="image" src="https://user-images.githubusercontent.com/35610748/114199514-a9341f00-9954-11eb-885c-631ce372375f.png">
it doesn't highlight for me 🤔

Reply:
🤔  let me check...